pkgs/by-name/sl/sly/update.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nix-shell
```

Comment:
you can copy the update logic of flet-client-flutter

this is the last bump try: https://r.ryantm.com/log/flet-client-flutter/2024-12-30.log

unfortunately the dependency thing breaks often on minors and I have to do manual intervention but it bumps fine with lockfile and stuff

Reply:
it breaks because I use it's src in the python packages as it's all in the same repo